src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -2303,15 +2256,13 @@ dpPrimTypeE ty =
         Float    -> wrap "Float"
         Double   -> wrap "Double"
 
-defaultImports ::
-  ( (?recordStyle :: RecordStyle)
-  , (?stringType :: StringType)
-  , (?typeLevelFormat :: Bool)
-  ) =>
-  -- | Uses GRPC?
-  Bool ->
-  [HsImportDecl]
-defaultImports icUsesGrpc | StringType stringModule stringType <- ?stringType =
+data ImportCustomisation = ImportCustomisation
+  { icStringType :: StringType
+  , icUsesGrpc :: Bool
+  }
+
+defaultImports :: ImportCustomisation -> [HsImportDecl]
+defaultImports ImportCustomisation{ icUsesGrpc, icStringType = StringType stringModule stringType} =
```

Comment:
This change should be reverted; it has nothing to do with large records and probably does not work.